client/components/Order.js

```diff
@@ -13,14 +13,32 @@ class Order extends React.Component {
 
   componentDidMount() {
     this.props.getHues()
+    this.filterHues()
   }
   // WILL ADD ADD ITEM AND REMOVE ITEM HERE AS WELL, IN ORDER TO ADJUST THE INPUT QUANTITIES FROM THE CART COMPONENT
 
+  addToCart(hue) {
+    // json.stringify method takes in an array/object, and setItem takes in two parameters - a key and a value. the value will be an array.
+    let localOrder = localStorage.getItem('order')
+    let order = []
+
+    // check to see if there are already items in the cart, or if this is a fresh cart.
+    if (localOrder) {
```

Comment:
I like how you handled updating the order in local storage! It makes a lot of sense.